src/components/repo-list/RepoList.js

```diff
@@ -24,15 +24,15 @@ const Repo = ({ repo, user }) => {
   ) : null;
 
   return (
-    <Box mb="12px">
+    <Box mb="12px" maxW="500px">
       <Flex align="baseline" justify="space-between">
         <Link as={RouterLink} to={`${user}/repo/${name}`}>
           <Text fontSize="24px">{name}</Text>
         </Link>
         {starsForDisplay}
       </Flex>
 
-      <Text fontSize="14px" mb="10px">
+      <Text fontSize="14px" mb="10px" isTruncated>
```

Comment:
We can always tweak the width later but this keeps it from overflowing. And the the truncation adds "..." if over one line

![Screen Shot 2021-01-11 at 3 02 52 PM](https://user-images.githubusercontent.com/39889198/104232271-1a2f6d80-541e-11eb-88ca-3ca77de73a12.png)